### PR TITLE
Feat/home screen

### PR DESCRIPTION
**Pull request by grzegorzkruk**
feat: add environment variables
feat: add navigation
feat: add home screen
feat: add weather store
feat: add http weather service
feat: add enum helper utilities

### CODE REVIEW

File: src/screens/HomeScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import React, {useCallback, useEffect, useMemo} from 'react';
+import {useWeatherStore} from '../store/weatherStore';
+import {CityWeather} from '../types/ListOfCitiesTypes';
+
+function HomeScreen() {
+  const cities = useWeatherStore(state => state.cities);
+  const loading = useWeatherStore(state => state.loading);
+  const fetchAllCities = useWeatherStore(state => state.fetchAllCities);
+
+  console.log('cities: ', cities);
```
Comment (adamTrz) [inline src/screens/HomeScreen.tsx:11]: 😉

File: src/screens/HomeScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  useEffect(() => {
+    try {
+      fetchAllCities();
+    } catch (e) {
+      console.log('fetch all cities error:', e);
```
Comment (adamTrz) [inline src/screens/HomeScreen.tsx:17]: Maybe worth adding some UI element as well so user know that there have been an error? Wdyt?

File: src/services/weatherService.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    baseURL: Config.WEATHER_API_URL,
+    timeout: 1000,
+    params: {
+      appId: Config.WEATHER_API_KEY,
+    },
+  });
+};
+
+export const getGroupWeather = (cityIds: AvailableCitiesCodes[]) => {
+  console.log('weatherService', weatherService().getUri());
```
Comment (adamTrz) [inline src/services/weatherService.ts:17]: ;)

**Review by adamTrz — APPROVED** [review]
LGTM! 💪 
Left one comment. Also, please remove console.logs ;)

File: src/screens/HomeScreen.tsx (hunk 2 above)
Comment (grzegorzkruk) [reply src/screens/HomeScreen.tsx:17]: Good point, I planned to add some toast along with UI implementation

**Pull request merged**